src/commands/exile_commands.py

```diff
@@ -93,3 +94,14 @@ async def roulette(interaction: discord.Interaction):
                     f"<@{interaction.user.id}> has tested their luck and lives another day...",
                     ephemeral=False,
                 )
+
+    @bot.tree.command()
+    @discord.app_commands.check(is_user_moderator)
+    @discord.app_commands.describe(user="User whose exile is being checked")
+    async def check_exile(interaction: discord.Interaction, user: discord.Member):
+        """Check the exile status of a user."""
+
+        async with create_logging_embed(interaction, user=user) as logging_embed:
+            msg = await get_user_exiles(logging_embed, user)
+
+            await interaction.response.send_message(msg, ephemeral=True)
```

Comment:
do we want this to be an ephemeral or an actual embedded message? @clairefleuret

Reply:
@brtran4 to make it as simple as possible, an ephemeral message makes the most sense. i dont think its required to send to the logging channel or report it there